### PR TITLE
Ezittel patch2

### PR DESCRIPTION
**Pull request by ezittel**
Completed work for   #714/ #715 
Added logo and Official Quest text to search details and icon to search.
Added awarded svg and text from database to details page and search result page.
Modified CSS for spacing issues.

### CODE REVIEW

File: app/components/views/Search.tsx [hunk 1]
```diff
@@ -328,6 +332,8 @@ export function renderDetails(props: SearchDetailsProps): JSX.Element {
         {(quest.ratingcount && quest.ratingcount >= 1) ? <StarRating readOnly={true} value={+ratingAvg} quantity={quest.ratingcount}/> : ''}
         <div className="indicators">
           {props.lastPlayed && <div className="lastPlayed"><DoneIcon className="inline_icon" /> Last played {Moment(props.lastPlayed).fromNow()}</div>}
+          {quest.official && <span> <img className="inline_icon" src="images/compass_small.svg"/> Official Quest!</span>}
+            {quest.awarded && <div className="awarded"><StarsIcon className="starsIcon" /> {quest.awarded} </div>}
```
Comment (toddmedema) [inline app/components/views/Search.tsx]: Looks like there's an unexpected extra indentation on this line?

Comment (ezittel) [reply app/components/views/Search.tsx]: I fixed this and will probably repull to get it all once done.

Comment (ezittel) [reply app/components/views/Search.tsx]: The overlap - I pointed that out, but I don't know how to fix it.  How do I float the indicators to the right?

Comment (ezittel) [reply app/components/views/Search.tsx]: Last Played/Awarded

image

Comment (ezittel) [reply app/components/views/Search.tsx]: Last Played + Official
image

**Comment by ezittel** [comment]
okay I got this fixed! I'm going to close this PR and start a new one with all of the changes

image

**Comment by toddmedema** [comment]
Note that you don't need to open a new PR every time you change the code - if you just make a new commit and push it to the same branch, the existing PR will be updated to reflect the changes :)

**Pull request closed without merging**